mapper/thin_edge_json/src/measurement.rs

```diff
@@ -0,0 +1,120 @@
+use chrono::offset::FixedOffset;
+use chrono::DateTime;
+/// The FlatMeasurementVisitor trait is used to group the measurements from descrete measurements.
+/// The descrete measurements might belong to a group or individual
+/// All these are grouped as one ThinEdge Json message
+/// For example the MeasurementGrouper implementing FlatMeasurementVisitor to group
+/// the messages
```

Comment:
I find this description not clear. I agree that this is not easy and will propose some text.